plugins/link/plugin.js

```diff
@@ -882,7 +882,7 @@
 		 */
 
 		/**
-		 * Optional JavaScript Regular Expression whenever telephone numbers in Link dialog should be validated.
+		 * Optional JavaScript Regular Expression whenever phone numbers in Link dialog should be validated.
 		 *
 		 *		config.linkTelNumberRegExp = /^[0-9]{9}$/;
```

Comment:
After this change, config variables have names using the old name of the field. What about changing them to `linkPhoneNumberRegExp` and `linkPhoneNumberMsg` or – even shorter – `linkPhoneRegExp` and `linkPhoneMsg`?

Reply:
That makes sense, I will refactor variables in the following commits.